prime-router/src/main/kotlin/cli/tests/SettingsTest.kt

```diff
@@ -78,7 +97,40 @@ class SettingsTest : CoolTest() {
 
     override suspend fun run(environment: Environment, options: CoolTestOptions): Boolean {
         ugly("Starting CRUD REST API ${environment.url}")
+        val bearer = getOktaAccessToken(environment, name)
+
+        val testCases = mutableListOf(
+            SettingsApiTestCase(
+                "simple history API happy path test",
+                "${environment.url}/api/waters/org/$historyTestOrgName/submissions",
+                emptyMap(),
+                listOf("pagesize" to options.submits),
+                bearer,
+                com.microsoft.azure.functions.HttpStatus.OK,
+            ),
+            SettingsApiTestCase(
+                "no such organization",
+                "${environment.url}/api/waters/org/gobblegobble/submissions",
+                emptyMap(),
+                listOf("pagesize" to options.submits),
+                bearer,
+                com.microsoft.azure.functions.HttpStatus.NOT_FOUND,
+            )
+        )
+        return runApiTestCases(testCases)
+    }
+
+    private fun runApiTestCases(testCases: List<SettingsApiTestCase>): Boolean {
+        val allPassed = testCases.map {
+            ugly("Starting test: ${it.name}")
+            val queryPass = runApiQuery(it)
+            if (queryPass) good("Test Passed:  ${it.name}")
+            queryPass
+        }.reduce { acc, onePassed -> acc and onePassed }
```

Comment:
This could be simplified to `.all { true }`

Reply:
true!